src/xdocs/config_imports.xml

```diff
@@ -603,11 +603,26 @@ import android.*;
           </tr>
           <tr>
             <td>illegalPkgs</td>
-            <td>packages to reject</td>
+            <td>Packages to reject, by default checks if import is the part of package. If <b>regexp</b> variable is set, then list of packages will be
+                interpreted as regular expressions. Note, all properties for match will be used. Order of matching: <b>illegalPkgs</b>, <b>illegalClasses</b></td>
```

Comment:
I made it because of @romani comment on previous changes
"please make a note in description that all properties for match will be used and describe order of matching to user."